src/UI/examples/Layout/Page/ui.php

```diff
@@ -0,0 +1,55 @@
+
+<?php
+include_once('z_auxilliary.php');
+
+function ui() {
+    global $DIC;
+    $f = $DIC->ui()->factory();
+    $renderer = $DIC->ui()->renderer();
+
+    $url = 'src/UI/examples/Layout/Page/ui.php?new_ui=1';
+    $btn = $f->button()->standard('See UI in fullscreen-mode', $url);
+    return $renderer->render($btn);
+}
+
+
+
+if ($_GET['new_ui'] == '1') {
+
+    chdir('../../../../../');
+
+    require_once("Services/Init/classes/class.ilInitialisation.php");
+
+    include_once('src/UI/examples/Layout/Metabar/metabar.php');
+    include_once('src/UI/examples/Layout/Page/page.php');
+    ilInitialisation::initILIAS();
```

Comment:
Vielleicht den Init-Foo noch in eine Extra-Funcktion auslagern und den Rendering-Foo auch. Dann kann man den Rendering-Foo drüberpacken und sieht das (leider nötige) Chaos nicht?